appservice/src/deploy/runWithZipStream.ts

```diff
@@ -80,6 +80,11 @@ async function getFilesFromGlob(folderPath: string): Promise<string[]> {
     const ignorePattern: string | string[] = zipDeployConfig.get<string | string[]>('zipIgnorePattern') || '';
     const filesToIgnore: string[] = await globby(ignorePattern, globOptions);
 
+    ext.outputChannel.appendLog(localize('zipIgnoreFilesMsg', 'Ignoring files from .vscode/settings.json \"appService.zipIgnorePattern\"'), { resourceName: client.fullName });
+    for (const file of filesToIgnore) {
+        ext.outputChannel.appendLine(localize('zipIgnoreFileNames', '\"{0}\"', file));
```

Comment:
No need to use `localize` here

Reply:
Do you mean for the lines or specifically the second one?
`    ext.outputChannel.appendLog(localize('zipIgnoreFilesMsg', 'Ignoring files from .vscode/settings.json \"appService.zipIgnorePattern\"'), { resourceName: client.fullName });
`

`ext.outputChannel.appendLine(localize('zipIgnoreFileNames', '\"{0}\"', file));`

Reply:
Specifically for "zipIgnoreFileNames"